lib/PrefsFeed.jsm

```diff
@@ -102,6 +102,10 @@ this.PrefsFeed = class PrefsFeed {
       "browser.newtabpage.activity-stream.discoverystream.spocs-endpoint",
       ""
     );
+    let discoveryStreamLangLayoutConfig = Services.prefs.getStringPref(
```

Comment:
The changes in this file just keep the page in sync with the pref value in case it changes.